src/graphql/data/nft/Collection.ts

```diff
@@ -1,10 +1,9 @@
 import graphql from 'babel-plugin-relay/macro'
 import { GenieCollection, Trait } from 'nft/types'
-import { useEffect } from 'react'
-import { useLazyLoadQuery, useQueryLoader } from 'react-relay'
 
-import { CollectionQuery } from './__generated__/CollectionQuery.graphql'
+import { useCollectionQueryQuery } from '../__generated__/types-and-hooks'
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const collectionQuery = graphql`
   query CollectionQuery($addresses: [String!]!) {
```

Comment:
probably more idiomatic to not add `Query` in the document node(, or remove the `Query` suffix from generated hooks so it doesn't read QueryQuery)

Reply:
One issue is that Relay requires the query to include `Query` as the suffix. The solution during this migration is to exclude this file from Relay's gql generation.